Scripts/components/Dialogs/CreateProject.cs

```diff
@@ -7,198 +7,202 @@ public class CreateProject : ReferenceRect
 {
 
 #region Signals
-    [Signal]
-    public delegate void project_created(ProjectFile projFile);
+	[Signal]
+	public delegate void project_created(ProjectFile projFile);
```

Comment:
The C# style guide says to use spaces, not tabs. You are replacing spaces with tabs here.

Reply:
Plus, I have made changes to CreateProject.cs/tscn and NewProject to support Godot 4 project files, so these need to be resolved.

Reply:
I didnt even realise lol. I will make the changes. Thanks!